libvast/vast/fbs/table_slice.fbs

```diff
@@ -32,6 +32,9 @@ table v1 {
 
   /// The Arrow Recod Batch containing all data.
   record_batch: [ubyte];
+
+  /// A timestamp assigned on import, representing a `vast::time` value.
```

Comment:
`representing nanoseconds since the unix epoch` may be more useful for the casual reader.